### DISCUSSION TITLE
21.08.02.월 회의록

### DISCUSSION THREAD

**Discussion by damilog** [Team-IT-DA/Frontend#28] [회의록]
- [x] global 상태 관리 전략(recoil atom 관리 어떻게 할 것인지? 페이지별 혹 기능별로 구분 필요성)
    - stores 폴더 하위에 각각의 페이지에 대한 타입파일을 생성해서 atom, selector 등을 관리한다.
    
![image](https://user-images.githubusercontent.com/65105537/127796146-79b8ccf0-01f4-4ddb-ab93-8ab79e593acc.png)

- [x] 남은 컴포넌트 뷰 담당구역(?) 나누기
    - ProductDetailPage : 제니가 구매수량 등 기본기능, UI 
    - ProductPage: 데이지가 Pagination, lazy loading, 검색결과 카운트 기능
    - SidebarPage: 디코가 header랑 붙이는 작업, 토글 기능, 장바구니 Material UI 적용, 회원가입 뷰 (시간 되면)
- [x] types 관리 전략
    - 중복되는 타입이 있을 수 있음. 모아놓을 것인지? 
    - 현재는 types라는 폴더가 있음. => 페이지별로 
    - 아래와 같은 방법으로 바꾼다.
    - `E` => enum, `T` => type, `I` => interface 를 앞에 붙이는 걸 컨벤션으로 한다.
![image](https://user-images.githubusercontent.com/65105537/127795343-edccbaa8-9e07-46a6-8fcc-d9382071bad2.png)

- [x] 오늘 결정한 상태관리 전략, 타입 관리 전략에 대해 wiki에 업데이트 

*회의 주제는 자유롭게 추가 해주세요!

**Comment by ha3158987** [comment]
BrandStory 페이지의 본문 줄바꿈을 현재는 line-height을 주는 걸로 처리를 해놓아서 원하는 곳에서 줄바꿈이 되지 않는 이슈가 있습니다. 
기획서와 다소 차이가 있는데 개선을 해야할지, 해야한다면 어떤 방법이 있을 지 생각해보면 좋을 것 같습니다! 

<기획서>
![image](https://user-images.githubusercontent.com/65105537/127782597-6862e65b-d7fc-4e97-80e4-77a372646839.png)



<실제 페이지> 
![image](https://user-images.githubusercontent.com/65105537/127782602-6def472d-d4ff-4cbb-82dc-3b33ab945c2b.png)

**Reply by ha3158987** [reply]
그냥 가는걸로!!
